vitrivr-engine-module-fes/build.gradle

```diff
@@ -0,0 +1,36 @@
+
+task generateFESClient(type: org.openapitools.generator.gradle.plugin.tasks.GenerateTask) {
+    generatorName = "kotlin"
+    inputSpec = "$rootDir/vitrivr-engine-module-fes/feature-extraction-server.json".toString()
+    outputDir = "$buildDir/api-client-generated".toString()
+    id = 'sample-client'
```

Comment:
Why is the id `sample-client` shouldn't this be something more meaningful?

Reply:
good catch, will fix